spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/ModuleDeploymentWriter.java

```diff
@@ -504,7 +482,7 @@ public synchronized void addResult(ModuleDeploymentStatus deploymentStatus) {
 								key.moduleDescriptorKey,
 								ModuleDeploymentStatus.State.failed,
 								String.format("Deployment of module '%s' to container '%s' timed out after %d ms",
-										key.moduleDescriptorKey, key.container, timeout)));
+										key.moduleDescriptorKey, key.container, deploymentTimeout)));
```

Comment:
Should be `deploymentTimeout.get()` because the string format is expecting a number.